src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        for (mate.academy.model.User user : users) {
+            if (email.equals(user.getEmail())) {
+                if (password.equals(user.getPassword())) {
+                    return true;
+                }
+            }
+        }
```

Comment:
we do not need to iterate over users in this method, we already have `findByEmail()` method that we can use here, and then just compare passwords and that's it, just do not forget about the null check :)